exercises/boolean-operators.md

```diff
@@ -47,27 +47,47 @@ The following exercises might be a bit dry, but the concept you'll be practising
 
     You can tell what the result will be without knowing the value of `x`. Why?
 
-3.  Write methods for the boolean operators `||`, `&&` and `!`. Here are the method signatures, just fill in the bodies:
+3.  Imagine you are developing an application that controls a cat door. There are 3 cats that you need to accommodate, Alpha, Bravo and Charlie.
+
+    a) Alpha is hurt and therefore shouldn't leave the house at the moment. Complete the following code to account for this:
```

Comment:
```suggestion
    a) Alpha is hurt and therefore shouldn't leave the house at the moment. Complete the following code so that Alpha can't leave the house:
```